tests/tests_pytorch/strategies/test_ddp_spawn.py

```diff
@@ -73,3 +77,14 @@ def test_ddp_all_dataloaders_passed_to_fit(tmpdir):
     )
     trainer.fit(model, train_dataloaders=model.train_dataloader(), val_dataloaders=model.val_dataloader())
     assert trainer.state.finished, "DDP doesn't work with dataloaders passed to fit()."
+
+
+def test_ddp_spawn_find_unused_parameters_exception():
+    """Test that the DDP strategy can change PyTorch's error message so that it's more useful for Lightning
+    users."""
+    # TODO: Update default
+    trainer = Trainer(accelerator="cpu", devices=1, strategy="ddp_spawn_find_unused_parameters_false", max_steps=2)
```

Comment:
TODO after https://github.com/Lightning-AI/lightning/pull/16611 gets merged
```suggestion
    trainer = Trainer(accelerator="cpu", devices=1, strategy="ddp", max_steps=2)
```